pom.xml

```diff
@@ -65,7 +65,8 @@
       <dependency>
         <groupId>com.datastax.cassandra</groupId>
         <artifactId>cassandra-driver-core</artifactId>
-        <version>3.0.3</version>
+        <version>3.1.2</version>
```

Comment:
I am fine with upgrading this to 3.1.x versions, but curious, what makes you decide to go with 3.1.x, instead of staying in the 3.0.x?